selenium/src/test/java/au/org/emii/portal/tests/step1/FacetUntickTest.java

```diff
@@ -23,6 +24,8 @@ public void untickFacetTest() throws InterruptedException {
         log.info("Loading search page - Step 1");
         WebDriver driver = getDriver();
         driver.get(AODN_PORTAL_SEARCH_PAGE);
+        WebDriverWait wait = new WebDriverWait(getDriver(), 30);
+        wait.withTimeout(10, TimeUnit.SECONDS);
```

Comment:
Whoops that method wasn't doing what I though it was. I've changed it to wait until there are results.